ui/admin/app/services/feature-edition.js

```diff
@@ -12,31 +11,43 @@ export default class FeatureEditionService extends Service {
 
   // =attributes
 
+  licensedFeatures = Object.keys(config.features.licensedFeatures);
```

Comment:
What is this `licensedFeatures` used for?

Reply:
https://github.com/hashicorp/boundary-ui/pull/1661/files?diff=split&w=0#diff-d2befbbcfc2fd4071e76ff3126f8e2cec284877fdf26740bddfa7b8b1f5727a9R103